b3lb/rest/endpoints/b3lb/endpoints.py

```diff
@@ -51,58 +51,111 @@
     "getDefaultConfigXML"
 ]
 
+ENDPOINTS_WITH_INTERNAL_ID = [
+    "create",
+    "join",
+    "end",
+    "isMeetingRunning",
+    "getMeetingInfo",
+    "setConfigXML"
+]
+
 LEGAL_ENDPOINTS = WHITELISTED_ENDPOINTS + BLACKLISTED_ENDPOINTS
+SUPPRESSED_PARAMS = ['dialNumber', 'voiceBridge']
 
 
 ##
 # Meta Routine
 ##
+async def api_pass_through(request, endpoint="", slug=None, sub_id=0):
+    """
+    Entrypoint for BBB APIs
+    """
+    # async: workaround for @require_http_methods decorator
+    if request.method not in ["GET", "POST"]:
+        return HttpResponseNotAllowed(["GET", "POST"])
+
+    parameters = request.GET
+
+    params = {}
+    for param in parameters:
+        params[param] = parameters[param]
+
+    if "checksum" in params:
+        checksum = params["checksum"]
+        del params["checksum"]
+    else:
+        return HttpResponse("Unauthorized", status=401)
```

Comment:
This should be a [HttpResponseBadRequest](https://docs.djangoproject.com/en/3.2/ref/request-response/#django.http.HttpResponseBadRequest) since checksum is a mandatory parameter, shouldn't it?

Reply:
BigBlueButton Servers send an XML with 400, so yes, BadRequest with content.